### PR TITLE
Remove tests on release

### PR DESCRIPTION
**Pull request by gerrod3**
[noissue]

### CODE REVIEW

File: templates/github/.github/workflows/release.yml.j2 [hunk 1]
```diff
@@ -115,8 +91,8 @@ jobs:
 
       {{ install_httpie() | indent(6) }}
 
-      {{ setup_env() | indent(6) }}
-
+      # Building the bindings and docs requires accessing the OpenAPI specs endpoint, so we need to
+      # setup the Pulp instance.
```
Comment (mdellweg) [inline templates/github/.github/workflows/release.yml.j2:95]: Actually, the content-app container should be sufficient... :thinking:

File: templates/github/.github/workflows/release.yml.j2 [hunk 2]
```diff
@@ -127,56 +103,55 @@ jobs:
...
+      {%- endif %}
+
+      - name: Build docs
+        run: |
+          pip install towncrier==19.9.0
+          towncrier --yes --version 4.0.0.ci
+          make -C docs/ PULP_URL="{{ pulp_scheme }}://pulp" diagrams html
+          tar -cvf docs/docs.tar docs/_build
 
-      {%- if docs_test %}
```
Comment (mdellweg) [inline templates/github/.github/workflows/release.yml.j2:165]: This is a plugin-template switch. I think for plugins that do not have/build docs we need to keep this, or consolidate with `publish_docs_to_pulpprojectdotorg`.

File: templates/github/.github/workflows/release.yml.j2 (hunk 1 above)
Comment (gerrod3) [reply templates/github/.github/workflows/release.yml.j2:95]: Do you mean the pulp-api app container?

Comment (mdellweg) [reply templates/github/.github/workflows/release.yml.j2:95]: Oh, sorry, unrelated. I was just thinking what the minimal requirement were to get the api.json. A container allowing to run pulpcore-manager should be sufficient, but in the ci we always fire up the complete single container. So please ignore!

**Review by mdellweg — APPROVED** [review]

**Pull request merged**